sites/athleticbusiness/server/templates/contact-us/index.marko

```diff
@@ -55,4 +55,8 @@ $ const query = {
     </div>
   </div>
 
+  <@below-container>
+    <endeavor-gam-ad-unit-define-display name="wa" out-of-page=true />
```

Comment:
You’ve already defined wa and reskin as “outOfPage” in the ad config... do you need to send that here?